Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -1027,6 +1027,12 @@ void UpdateToolbar()
 
 			bar.Title = currentPage.Title ?? "";
 
+			if (_toolbar.NavigationIcon != null && !textColor.IsDefault)
+			{
+				var drawable = (_toolbar.NavigationIcon as DrawerArrowDrawable);
```

Comment:
Just include a null validation like here: https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs#L502

```
var icon = _toolbar.NavigationIcon as DrawerArrowDrawable;
if (icon == null)
     return;
```

Reply:
@jsuarezruiz, why should it return from there? I understand having the null check but I don't understand the return because we still have some function calls after this `if` block like `UpdateTitleView()`.

Reply:
Right, is necessary the null check but not the return (I see that we updated the TitleView below).